README.md

```diff
@@ -135,6 +136,14 @@ if numba_mpi.rank() == 0:
 
 ![plot](https://github.com/numba-mpi/numba-mpi/releases/download/tip/readme_plot.png)
 
+### Known Issues
+
+**NOTE**: Issues listed below only relate to combinations of platforms and MPI distributions that we target to support, but due to various reason are currently not working and are temporarily excluded from automated testing:
+
+- tests on Ubuntu 2024.4 that use MPICH are not run due to failures caused by newer version of MPICH (`4.2.0`); note, that previous tests ran
+using version `4.0.2` of MPICH (that is installed by default on Ubuntu 2022.4 using `apt`) were passing (see [related issue](https://github.com/numba-mpi/numba-mpi/issues/162)),
+- tests on Intel MacOS (v13) that use OpenMPI are currently not run due to failures being under investigation (see [related issue](https://github.com/numba-mpi/numba-mpi/issues/163)),
```

Comment:
here, we could insert "TODO #163" so that CI will force us to remove that item in case the issue is closed

Reply:
(i.e., it will do so after #107 is addressed)

Reply:
done